packages/block-library/src/latest-comments/style.scss

```diff
@@ -1,4 +1,15 @@
+// Lower specificity - target list element.
 ol.wp-block-latest-comments {
+	// Removes left spacing in Customizer Widgets screen.
+	// Due to low specificity this will be safely overriden
+	// by default wp-block layout styles in the Post/Site editor
+	margin-left: 0;
```

Comment:
My biggest concern was this one, exactly because of the auto margins for centering. Thanks for adding the comment to explicitly address taht.

Reply:
> adding the comment to explicitly address that.

Exactly this. I'm a big proponent of not adding random CSS rules without explaining _why_ they are needed.

Why? Because otherwise things can get easily removed without folks understanding the consequences 😄